marathon/marathon_stub.go

```diff
@@ -4,46 +4,56 @@ import (
 	"errors"
 
 	"github.com/allegro/marathon-consul/apps"
+	"sync"
 )
 
 type MarathonerStub struct {
-	AppsStub     []*apps.App
-	AppStub      map[apps.AppID]*apps.App
-	TasksStub    map[apps.AppID][]*apps.Task
-	leader       string
-	interactions bool
+	AppsStub       []*apps.App
+	AppStub        map[apps.AppID]*apps.App
+	TasksStub      map[apps.AppID][]*apps.Task
+	leader         string
+	interactionsMu sync.RWMutex
+	interactions   bool
```

Comment:
I would rather leave it with mutex for simplicity/readability sake - no need for "smart" code here, this is just test stub (not hot part of code) and from what i see bool is enough. We can add counter when/If we need it.